docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -26,14 +26,14 @@ By default, deprecation warnings are logged when the log level is set to `WARN`.
 . Review the <<elastic-stack-breaking-changes>> and upgrade your code to work
 with {version}.
 
-. Upgrade to 6.7 and use the {kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to {ref}/docs-reindex.html[reindex]
+. Upgrade to 6.8 and use the {kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to {ref}/docs-reindex.html[reindex]
 any indices that are not compatible with {version}.
 +
 [role="xpack"]
 .Upgrade Assistant
 ******
 The Upgrade Assistant and migration APIs are enabled with both the Basic and
-Trial licenses. You can install the default distribution of 6.7 to use the
+Trial licenses. You can install the default distribution of 6.8 to use the
```

Comment:
Is the deletion of trial licenses relevant here?

Reply:
From what I understand, trial licenses can still be used for machine learning features.